doc_admin/AdminManual.adoc

```diff
@@ -49,50 +50,105 @@ Open Hospital features include:
 * Internal communication
 * Statistics and printing
 
-In the following chapters, all the information needed to install, configure, run and maintain an Open Hospital installation are presented.
-The information included also allows enabling and disabling functionality as well as managing users and groups.
-All the information needed to use the application software is not included; please refer to the _User’s Guide_, supplied with the application software.
+=== 1.2 Hardware requirements
 
-<<<
+Minimum hardware requirements for running Open Hospital:
 
-== 3 Installation
+* PC / Notebook with dual core CPU
+* 2 Gb RAM (4 Gb recommended)
+* 4 Gb free hard disk space
 
-=== 3.1 Open Hospital
+=== 1.3 Software requirements
 
-Open Hospital is a client application that can be used on a single computer (PORTABLE mode) or in a client/server configuration (CLIENT mode). The server part utilizes a DBMSfootnote:[DataBase Management System] to archive, share, and present the information. The free and open-source MariaDBfootnote:[See https://www.mariadb.org] is the database server of choice.
+Minimum Operating System version required:
 
-The typical client/server, networked (LAN) configuration is shown in the following figure:
+* Windows 7 (Windows 10/11 recommended)
+* Ubuntu 18.04 (Ubuntu 20.04 recommended)
 
-image:image3.png[image,width=642,height=243]
+32 bit (i686) and 64 bit (x86_64) architectures are supported.
 
-The Administrator/Installer tasks are hereby listed:
+[#download]
+=== 1.5 Download
+
+Open Hospital official releases can be downloaded at the following link:
+
+**Download latest release from github**: https://github.com/informatici/openhospital/releases/latest
+
+More information can be found on the Open Hospital website: https://www.open-hospital.org/download
 
-* Install the JRE - Java Runtime Environment (see compatibility matrix for the right version) on every client
-* Install the MariaDB/MySQL server on a computer that will act as the OH server
-* Create the OH database through the execution of a script in the folder sql/ within the .zip file folders
-* Copy the Open Hospital application on every client (it may be installed on the server also)
-* Configure each Open Hospital instance to point to the OH server (using the hostname or the IP address)
+[#setup]
+=== 1.6 Initial setup
 
-For the Java JRE installation refer to the next chapter <<jre,Java Runtime Environment>>
-and the online documentation.
+Open Hospital does not require installation. It is sufficient to unzip the downloaded package in a folder of choice, e.g.
 
-For the MariaDB/MySQL database server installation and configuration refer to the chapter <<database-server,Database Server>> and the online documentation.
+* on Linux:
+----
+/home/OH/OpenHospital-[VERSION]		# Linux
+C:\Users\OH\OpenHospital-[VERSION]	# Windows
```

Comment:
This is confusing; providing the windows path under a bullet point that reads `on Linux`

Reply:
Fixed.